src/bp/nlu-server/api.ts

```diff
@@ -153,17 +155,28 @@ export default async function(options: APIOptions, nluVersion: string) {
   router.post('/predict/:modelId', async (req, res) => {
     try {
       const { modelId } = req.params
-      const { sentence, password } = req.body
+      const { texts, password } = req.body
+
+      if (options.batchSize && texts.length > options.batchSize) {
```

Comment:
That might be me being ADHD but as a user, I would expect 100 to be allowed if I define a batchSize of 100.

`>=`

Reply:
yeah it is allowed. If --batchSize===100 and texts.length === 100 the condition is false and no error is thrown

Reply:
gosh damn I'm sorry I didn't read carefully .